grc/gui/PropsDialog.py

```diff
@@ -171,7 +171,7 @@ def update_gui(self, widget=None, force=False):
 
                 for param in self._block.params.values():
                     if force_show_id and param.dtype == 'id':
-                        param.hide = 'none'
+                        param.hide = 'part'
```

Comment:
This (permanently) modifies the param object (as does the current version). I find this highly troublesome.

Reply:
I had thought this initially as well, but I could not find any actual issue; selecting the toggle overrides the PropsDialog ID behavior to part/none and deselecting uses the yaml behavior. My change to 'part' here actually does nothing as none/part have the same behavior for the PropsDialog.